subprojects/distributions-dependencies/build.gradle.kts

```diff
@@ -178,4 +178,41 @@ dependencies {
         api(libs.xerces)                { version { strictly("2.12.0") }}
         api(libs.xmlunit)               { version { strictly("1.6") }}
     }
+
+    components {
```

Comment:
I do not think this is required either. The replacement and conflict detection should be handled elsewhere.

Reply:
okay, I removed the 'jsch replacement' rule (and in other places)